README.adoc

```diff
@@ -0,0 +1,156 @@
+:context: rhoas-cli-command-reference
```

Comment:
@bhardesty Let me know how this works for you. If you want to have quick meeting I'm happy to chat about this